src/net.h

```diff
@@ -113,6 +113,14 @@ struct CSerializedNetMsg
     std::string command;
 };
 
+enum class ConnectionType {
```

Comment:
Consider using doxygen-style comments for these enum entries

Reply:
done